react-app/src/components/layout/Header.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import headerImageLarge from '@/assets/images/BCID_H_rgb_pos.png';
+import headerImageSmall from '@/assets/images/BCID_V_rgb_pos.png';
+import { AppBar, Link, Box, Button, Divider, Toolbar, Typography, useTheme } from '@mui/material';
+import { Link as RouterLink } from 'react-router-dom';
+import { useKeycloak } from '@bcgov/citz-imb-kc-react';
+
+const AppBrand = () => {
+  const theme = useTheme();
+  return (
+    <Box
+      sx={{
+        '& a': {
+          display: 'flex',
+          alignItems: 'center',
+          overflow: 'hidden',
+          textDecoration: 'none',
+        },
+        '& img': {
+          mr: 2,
+        },
+      }}
+    >
+      <RouterLink to="/" aria-label="Go to PIMS Home">
+        <Box>
+          <picture>
+            <source srcSet={headerImageLarge} media="(min-width: 960px)"></source>
+            <source srcSet={headerImageSmall} media="(min-width: 600px)"></source>
+            <img
+              src={headerImageSmall}
+              alt={'Government of British Columbia'}
+              style={{ height: '48px' }}
+            />
+          </picture>
+        </Box>
+        <Box>
+          <Divider
+            style={{ borderBottomWidth: '26px', borderRightWidth: '2px' }}
+            orientation="vertical"
+          />
+        </Box>
```

Comment:
Does a Divider need a Box if it's the only child?

Reply:
It doesn't, not sure why I put those there. Removed.